api_testrail.py

```diff
@@ -150,11 +173,12 @@ def testrail_project_ids(self, project):
            from each respective project
         """
         # Query with filtering
-        q = self.db.session.query(Projects).filter(Projects.project_name_abbrev.in_(project)) # noqa
+        q = self.db.session.query(Projects).filter(Projects.project_name_abbrev == project)  # noqa
```

Comment:
This change is breaking our calls when we don't specify one project but we iterate over all mobile projects: 
`python3 ./__main__.py --platform mobile --project ALL --report-type testrail-milestones`

If you run: 
`python3 ./__main__.py --platform mobile --project fenix --report-type testrail-milestones` works but we need it to work for one project or all